docs/dev/konnector.md

```diff
@@ -42,24 +41,11 @@ Then, write your code into `konnectors.js` and build the application, running `y
 
 ### Collector structure
 
-Basically, a collector is just an object with some required fields:
+Basically, a connector is just an object passed to baseKonnector.createNew:
 
- - `name`: the displayed name of the collector;
```

Comment:
Could you please move this attributes descriptions into the section describing the Manifest?